pkg/skaffold/schema/latest/config.go

```diff
@@ -254,6 +254,9 @@ type ClusterDetails struct {
 
 	// DockerConfig describes how to mount the local Docker configuration into a pod.
 	DockerConfig *DockerConfig `yaml:"dockerConfig,omitempty"`
+
```

Comment:
Those changes will require introducing a `v1beta8` schema

Reply:
@dgageot should the new schema version be introduced as part of this change or separately?

Reply:
actually, ignore that comment, I can see that a new schema version has already been introduced.